Apps/Sandcastle/gallery/Clamp to 3D Tiles.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Clamp a model to a 3D Tileset using the sampleHeight function.">
+    <meta name="cesium-sandcastle-labels" content="Showcases, 3D Tiles">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === 'function') {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    terrainProvider : Cesium.createWorldTerrain()
+});
+var scene = viewer.scene;
+
+if (!scene.clampToHeightSupported) {
+    console.log('This browser does not support clampToHeight.');
```

Comment:
The only thing that happens is Sandcastle gets stuck on the loading screen if returning early. I think it's preferable to try to avoid that, but maybe it doesn't matter if the demo wouldn't work anyways.

<img width="452" alt="loading" src="https://user-images.githubusercontent.com/915398/45640942-0360f200-ba82-11e8-8a07-8cbd8a70d7e5.PNG">

Reply:
OK, let's keep it so the example doesn't get stuck.